src/converter/entities.ts

```diff
@@ -0,0 +1,93 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
With the Opaque types, this can be avoided, which is generally desirable because where we would want types, we're not encouraged to have them (such as `IntegrationInstance` as an argument to some of these converters).